File: wiringX/blink.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include <ctype.h>
#include <unistd.h>
#include "wiringx.h"

#define gpio 8

int main(int argc, char *argv[]) {

	if(wiringXSetup("rock4", NULL) == -1) {
		wiringXGC();
		return -1;
	}
	pinMode(gpio, PINMODE_OUTPUT);

	while(1) {
		digitalWrite(gpio, HIGH);
		sleep(1);
		digitalWrite(gpio, LOW);
		sleep(1);
	}

	return 0;
}

File: wiringX/python/blink.py
#!/usr/bin/env python3
# -*- encoding: utf-8 -*-

import os
import sys
from time import sleep
from wiringX import gpio

gpio.setup(gpio.ROCK4);

gpio.pinMode(gpio.PIN8, gpio.PINMODE_OUTPUT);

try:
	while True:
		gpio.digitalWrite(gpio.PIN8, gpio.HIGH);
		sleep(1);
		gpio.digitalWrite(gpio.PIN8, gpio.LOW);
		sleep(1);
except KeyboardInterrupt:
	pass
